### ISSUE TITLE
Website is unusable

### ISSUE THREAD

**Issue by ComradeSparklePony** [eustasy/midori-browser.org#110]
I went to the Midori Browser website earlier today, and although I made it to the website, it was a single page saying that the website was down.

I came back later, and it appeared to be back up. I scrolled to read more about the browser, and came across the following text:

> Analytics release series A financing launch party interaction design android angel investor. Vesting period social media sales conversion launch party rockstar incubator customer social proof graphical user interface technology supply chain. Non-disclosure agreement rockstar creative partner network launch party social proof paradigm shift long tail influencer.

Looking at the most recent snapshot of the website on the [Wayback Machine (November 20, 2018)](https://web.archive.org/web/20181115033907/https://www.midori-browser.org/), this did not seem right.

Further problems with the website include:

- Download button scrolls page down to "Neve" section, which does not tell me what "Neve" is.
- Features button scrolls down to "features" section, which also has a bunch of random-seeming words.
- Social media links at the bottom of the page just link to the midori-browser.org
- Video plays an ad for the OrbitFox WordPress plugin.

Here are some screenshots:
![screen shot 2019-01-10 at 3 00 52 pm](https://user-images.githubusercontent.com/27787087/50993995-05dfcf80-14e9-11e9-8b52-fd381dec54bb.png)
![screen shot 2019-01-10 at 3 01 14 pm](https://user-images.githubusercontent.com/27787087/50993996-05dfcf80-14e9-11e9-8175-600f2a32af32.png)

In conclusion, the Midori browser website seems a lot sketchier than when I was on it last. Is this intentional, or am I being paranoid?

Thank you for reading all this.

**Comment by ComradeSparklePony** [comment]
Seems mostly fixed now. The video still is the WordPress plugin ad though.
